java/src/jmri/jmrit/display/layoutEditor/LayoutTrack.java

```diff
@@ -106,5 +101,24 @@ public void setHidden(boolean hide) {
         hidden = hide;
     }
 
+    /**
+     * non-accessor methods
+     */
+
+    //NOTE: not public because "center" is a member variable
```

Comment:
What does the ownership of center have to do with this not being public. Many public methods act on non-public fields throughout JMRI.